src/rastervision/samples/workflow-configs/segmentation/deeplab-test-remote.json

```diff
@@ -0,0 +1,94 @@
+{
+    "train_scenes": [
+        {
+            "id": "2-10",
+            "raster_source": {
+                "geotiff_files": {
+                    "uris": [
+                        "/opt/data/top_potsdam_2_10_RGBIR.tif"
+                    ]
+                }
+            },
+            "ground_truth_label_store": {
+                "segmentation_raster_file": {
+                    "src": {
+                        "geotiff_files": {
+                            "uris": [
+                                "/opt/data/top_potsdam_2_10_label_georeferenced.tif"
+                            ]
+                        }
+                    },
+                    "src_classes": [ "#ffff00", "#0000ff" ],
+                    "dst_classes": [ 1, 0 ]
+                }
+            }
+        }
+    ],
+    "test_scenes": [
+        {
+            "id": "2-11",
+            "raster_source": {
+                "geotiff_files": {
+                    "uris": [
+                        "/opt/data/top_potsdam_2_11_RGBIR.tif"
+                    ]
+                }
+            },
+            "ground_truth_label_store": {
+                "segmentation_raster_file": {
+                    "src": {
+                        "geotiff_files": {
+                            "uris": [
+                                "/opt/data/top_potsdam_2_11_label_georeferenced.tif"
+                            ]
+                        }
+                    },
+                    "src_classes": [ "#ffff00", "#0000ff" ],
+                    "dst_classes": [ 1, 0 ]
```

Comment:
Why are there two elements in `dst_classes` but only one in `class_items`? I think it has something to do with the fact that everything that's not in the `class_map` counts as background and is assigned the id 0. But if that's true, then I still don't understand why 0 needs to be mapped to a hex value at all. Perhaps there should be more documentation about converting classes to the background value.

Reply:
So we are training a model to distinguish between car and building. But, then shouldn't there be an item in `class_items` for building?

Reply:
There is only one class under consideration: cars.  Let me make some changes to clarify the situation.